OneCommunityGlobal/HighestGoodNetworkApp
# Lucas fix another user dashboard

# Description
(PRIORITY MEDIUM) Jae: Fix viewing another person’s dashboard view of header  (WIP Lucas)
Leaderboard → Click dot by person’s name
This view should show the other person’s dashboard completely… right now it still shows the header like the viewer of the dashboard. I want an anyone who clicks those dots to see exactly what they see
The exception should be that non Admin/Owner accounts shouldn’t be able to interact with the other person’s header options. An Admin/Owner account SHOULD be able to though… as if they were that other person. 
Need this for diagnosing problems


## Main changes explained:
Now, when viewing another user's dashboard, the view of the header will match what the user would see with some minor differences.

1. Owner/administrator users viewing another user's dashboard will be able to interact with the timer, reset password, visit the timelog window and other things.
2. Non Owner/administrators should only be able to see the header, the timelog page and the profile page
3. On the timelog page, on the tasks tab, the tasks should match the viewed user's tasks
4. When interacting with the timer, the tasks/projects should match the viewed user's tasks/projects

## How to test:
1. check into current branch
5. do `npm install` and `...` to run this PR locally
6. log as Owner/administrator user
7. On the Leaderboard, click on the dot on the left side of the user's name
8. See if the header matches the viewed user's header with the explicitated cases
9. log as non Owner/administrator user
10. See if the header matches the viewed user's headers with the explicitated cases

## Note:
Please make sure to see if everyting works as it should on the authenticated user's dashboard and on the viewed user's dashboard.
